src/app/lib/config/services/russian.js

```diff
@@ -44,8 +44,9 @@ export const service = {
     themeColor: `${C_POSTBOX}`,
     twitterCreator: '@bbcrussian',
     twitterSite: '@bbcrussian',
-    noBylinesPolicy: 'https://www.bbc.com/news/help-41670342#authorexpertise',
-    publishingPrinciples: 'https://www.bbc.com/news/help-41670342',
+    noBylinesPolicy:
+      'https://www.bbc.com/russian/institutional-50098149#authorexpertise',
+    publishingPrinciples: 'https://www.bbc.com/russian/institutional-50098149',
```

Comment:
@OlgaLyubin all the new links in the Russian service redirect to 404 pages, is that expected or should the relevant articles be published first?

Reply:
They should be published first. I will add a "do not merge" to this PR's name and will not merge it before all the links are working:)

Reply:
@OlgaLyubin Might be good to add a note to the description as well just for visibility.